src/main/java/hudson/plugins/dry/DryPublisher.java

```diff
@@ -35,105 +37,19 @@
 
     private static final String DEFAULT_DRY_PATTERN = "**/cpd.xml";
     /** Ant file-set pattern of files to work with. */
-    private final String pattern;
+    private String pattern;
 
     /** Minimum number of duplicate lines for high priority warnings. @since 2.5 */
-    private final int highThreshold;
+    private int highThreshold;
     /** Minimum number of duplicate lines for normal priority warnings. @since 2.5 */
-    private final int normalThreshold;
+    private int normalThreshold;
```

Comment:
Why? You did not make these into optional parameters. (Though it seems they ought to be, especially if they were introduced later—presumably there is some sensible default.)